dist/angular-input-fields.js

```diff
@@ -179,8 +179,22 @@
         }
     ]);
 
+    angular.module('aif-add-suffix-filter', []).filter('aifAddSuffix', [
+        function () {
+            return _.memoize(function (input, suffix) {
+                if (angular.isUndefined(suffix)) {
+                    suffix = '';
+                }
+                return input + '' + suffix;
+            }, function (input, suffix) {
+                return input + '' + suffix;
```

Comment:
Don't need `''` in the middle here. Will a `+` operation ever end up with `null`, `undefined`, `NaN`, `true`, `false`, `Infinity`, or `-Infinity`? If so, then keep the `''`, otherwise it doesn't matter